src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -79,6 +79,15 @@ void uct_rc_mlx5_iface_check_rx_completion(uct_rc_mlx5_iface_common_t *iface,
 
     ucs_memory_cpu_load_fence();
 
+    if (uct_ib_mlx5_cqe_is_zipped(cqe->op_own)) {
+        if (cq->cq_unzip.left > 0) {
+            return uct_ib_mlx5_iface_cqe_unzip_continue(cq);
+        }
+        return uct_ib_mlx5_iface_cqe_unzip_start(cqe, cq);
```

Comment:
IMO it's not good style that 'start' calls 'continue'. it's mixing different operations in one function.
Also, if remaining (left) >0, we don't need to check again of CQE is compressed. this is the more common case since it happens for every mini-cqe.

better like this:
```c
if (cq->cq_unzip.remaining > 0) {
    return uct_ib_mlx5_iface_cqe_unzip_continue(cq);
} else if (uct_ib_mlx5_cqe_is_zipped(cqe->op_own)) {
    /* First zipped CQE in the sequence */
    uct_ib_mlx5_iface_cqe_unzip_start(cqe, cq);
    return uct_ib_mlx5_iface_cqe_unzip_continue(cq);
}

```